java/android/nnstreamer/src/main/jni/Android.mk

```diff
@@ -58,6 +58,13 @@ _ENABLE_TF_LITE := false
 endif #endif ($(filter $(TARGET_ARCH_ABI), arm64-v8a x86_64),)
 endif #endif ($(ENABLE_TF_LITE),true)
 
+# TFLite QNN Delegate
+ENABLE_TF_LITE_QNN_DELEGATE := false
```

Comment:
Q: difference between ENABLE_TF_LITE_QNN_DELEGATE and TFLITE_ENABLE_QNN_DELEGATE

is 'ENABLE_TF_LITE_QNN_DELEGATE' necessary?

Reply:
Thanks! It's redundant and misleading. I removed `ENABLE_TF_LITE_QNN_DELEGATE`, PTAL